letsencrypt/models/letsencrypt.py

```diff
@@ -74,12 +75,20 @@ def generate_domain_key(self, domain):
 
     @api.model
     def validate_domain(self, domain):
-        if domain in ['localhost', '127.0.0.1'] or\
-                domain.startswith('10.') or\
-                domain.startswith('172.16.') or\
-                domain.startswith('192.168.'):
+        local_domains = ['localhost', 'localhost.localdomain']
```

Comment:
A simple `cat /etc/hosts` reveals that you should add `localhost6.localdomain6` and `localhost6`, please.